src/compiler/output-targets/test/output-targets-dist-custom-elements.spec.ts

```diff
@@ -111,6 +112,21 @@ describe('Custom Elements output target', () => {
     });
   });
 
+  describe('bundleCustomElements', () => {
+    it('should set a diagnostic if no `dir` prop on the output target', async () => {
+      const { config, compilerCtx, buildCtx } = setup();
+      const outputTarget: OutputTargetDistCustomElements = { type: 'dist-custom-elements' };
+      const ret = await bundleCustomElements(config, compilerCtx, buildCtx, outputTarget);
```

Comment:
Hmmmm does this fail lint? I don't think this is being used ATM 🤔 

I'm validating a fix for the linter now (https://github.com/ionic-team/stencil/pull/3376). In the mean time can we remove this?

```ts
await bundleCustomElements(config, compilerCtx, buildCtx, outputTarget);
```

Reply:
ahhh that's my bad, I'm using `zsh` and `npx eslint src/*.ts src/**/*.ts` returns errors there, but the same command in `package.json` doesn't (I think because it uses `sh`) 😢